src/utils/update/review.beneficial.owner.ts

```diff
@@ -0,0 +1,96 @@
+import { BeneficialOwnerGov } from "../../model/beneficial.owner.gov.model";
+import * as config from "../../config";
+import { ApplicationData } from "../../model";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
+
+const AllBoTypes = {
+  boiReview: "review_beneficial_owners_individual",
+  boGovReview: "review_beneficial_owners_government_or_public_authority",
+  boIndividual: "beneficial_owners_individual",
+  boGov: "beneficial_owners_government_or_public_authority"
+};
+
+const beneficialOwnerReviewRedirectUrl = `${config.UPDATE_AN_OVERSEAS_ENTITY_URL
+      + config.UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE
+      + config.REVIEW_BENEFICIAL_OWNER_INDEX_PARAM}`;
+
+const beneficialOwnerGovReviewRedirectUrl = `${config.UPDATE_AN_OVERSEAS_ENTITY_URL
+        + config.UPDATE_REVIEW_BENEFICIAL_OWNER_GOV_PAGE
+        + config.REVIEW_BENEFICIAL_OWNER_INDEX_PARAM}`;
+
+const checkBoIndividualValidation = (boi: BeneficialOwnerIndividual): boolean => {
+  return boi?.usual_residential_address ? true : false;
+};
+
+const checkBoGovValidation = (boi: BeneficialOwnerGov): boolean => {
+  return boi?.principal_address ? true : false;
```

Comment:
'bog' 😆  instead of 'boi' here? What are these validations checking for, is something similar needed for BO other?

Reply:
Fixed